packages/app-store/routing-forms/trpc/deleteForm.handler.ts

```diff
@@ -1,6 +1,5 @@
-import type { PrismaClient } from "@prisma/client";
-
 import { entityPrismaWhereClause } from "@calcom/lib/entityPermissionUtils";
+import type { PrismaType } from "@calcom/prisma";
```

Comment:
Made these changes to support Prisma Client extensions. We anyway should have read the prisma type from @calcom/prisma only.